config/dbaasprovider/dbaas_provider.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: DBaaSProvider
+metadata:
+  name: mongodb-atlas-registration
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+spec:
+  provider:
+    name: Red Hat DBaaS / MongoDB Atlas
+    displayName: MongoDB Atlas Cloud Database Service
+    displayDescription: MongoDB Atlas Fully Managed Database Service.
```

Comment:
not an issue, just to make sure to discuss this with Mondo DB if the display description is the right one or you can have a look on their website if required any change in that. 
https://www.mongodb.com/cloud/atlas/lp/try2?utm_source=google&utm_campaign=gs_americas_united_states_search_core_brand_atlas_desktop&utm_term=mongodb%20atlas%20login&utm_medium=cpc_paid_search&utm_ad=e&utm_ad_campaign_id=12212624338&gclid=EAIaIQobChMI7efI4PyK8gIVI4VaBR3tOAHmEAAYASAAEgIMQvD_BwE

Reply:
This one is better :). Change to "Cloud-hosted MongoDB service on AWS, Azure and Google Cloud"